include/receiver_cadmium.hpp

```diff
@@ -0,0 +1,116 @@
+/**
+* Cristina Ruiz Martin
+* ARSLab - Carleton University
+*
+*/
+
+#ifndef _BOOST_SIMULATION_PDEVS_RECEIVER_HPP_
```

Comment:
In each file according to rule 13. have header guards professor has given 2 guards at end and beginning. You have used only 1. I am not sure single is fine or we need double guards. Just noticed so mentioned in the review comments.
example: 
#ifndef __MY_HEADER__
#define __MY_HEADER__
... #endif // __ MY_HEADER__

Reply:
I am also not sure regarding it. I looked it up online and found mixed opinions. Anyways, I added an additional underscore with all of them.